components/centraldashboard/app/server.ts

```diff
@@ -52,6 +59,16 @@ async function main() {
 
   console.info(`Using Profiles service at ${profilesServiceUrl}`);
   const profilesService = new DefaultApi(profilesServiceUrl);
+  const metricsPort: number = Number(METRICS_PORT);
+
+  // Custom metrics configuration
+  app.use(
+    responseTime((req: Request, res: Response, time: number) => {
+      restHttpRequestTotal.labels({ method: req.method, status: res.statusCode }).inc();
+      restHttpRequestDuration.labels(
+        { method: req.method, path: req.baseUrl, status: res.statusCode }).observe(time);
+    }),
+  );
```

Comment:
Would it make sense to enable/disable the feature according to env variables, similar to the crud-web-apps? And then add the required environment variable in this deployment